view/theme/frio/templates/admin/queue.tpl

```diff
@@ -0,0 +1,25 @@
+<div id='adminpage'>
```

Comment:
Please use double quotes for HTML attributes.

Reply:
I don't want to argue with you, but you should checkout the original (view/templates/admin/queue.tpl)

I only added the class table to the the tables ;)